src/test/java/org/gitlab4j/api/TestUtils.java

```diff
@@ -33,8 +32,7 @@ public static String getReaderContentAsString(Reader reader) throws IOException
     static {
 
         testProperties = new Properties();
-        File path = new File(System.getProperty("user.home"), "test-gitlab4j.properties");
```

Comment:
https://stackoverflow.com/questions/5529532/how-to-get-a-test-resource-file

To keep the resource under the test directory of the project. If this is undesired, I can remove it.

Reply:
A few things here:

1. Not a fan of having it down in a src/test/resource directory, it leads to the potential of secrets being accidentally checked in.  I would prefer it in the root of the project, and name it:
```
example-test-gitlab4j.properties
```

When moved to the root the code in TestUtils would need to look something like:
```
try (InputStream input = new FileInputStream("test-gitlab4j.properties")) {

```

2. Include test-gitlab4j.properties in the .gitignore file so it is not accidentally checked in.

3.  There are about 10 additional properties that need to be set to fully test things, I'll get you those when I have some time tomorrow afternoon.

Last but not least.  I am working on a GitLab server docker image that will be used to test GitLab4J, this way the test-gitlab4j.properties file can actually be created and left as is.  I expect to have this finished and checked in within the next 2-3 weeks.